himan-lib/source/ensemble.cpp

```diff
@@ -189,9 +189,47 @@ std::vector<float> ensemble::Values() const
 
 std::vector<float> ensemble::SortedValues() const
 {
-	std::vector<float> v = RemoveMissingValues(Values());
-	std::sort(v.begin(), v.end());
-	return v;
+	return SortedValues(kRemove);
+}
+
+std::vector<float> ensemble::SortedValues(const HPMissingValueTreatment treatMissing) const
+{
+	std::vector<float> res;
+	switch (treatMissing)
+	{
```

Comment:
Maybe add a brief description why this is done including an example of a parameter that it's done.